clients/drcachesim/scheduler/scheduler_impl.h

```diff
@@ -555,8 +555,9 @@ template <typename RecordType, typename ReaderType> class scheduler_impl_tmpl_t
         uint64_t input_count = 0;
     };
 
-    // We assume a 2GHz clock and IPC=1.
-    static constexpr uint64_t INSTRS_PER_US = 2000;
+    // We assume a 2GHz clock and IPC=0.5 to match
+    // scheduler_options_t.time_units_per_us.
```

Comment:
Can we have a change-detector test to make sure the two values stay the same?

(Maybe by creating a default scheduler_options_t and comparing it with scheduler_impl_tmpl_t::INSTRS_PER_US inside a test sub-class of scheduler_impl_tmpl_t; this way we won't have to expand the visibility of INSTRS_PER_US)

Reply:
Done: added such a test to scheduler_unit_tests.